webcompat/templates/web_modules/milestone-editor.jst

```diff
@@ -14,7 +14,7 @@
           <input class="wc-CategoryEditor-list-item-checkbox"
                  type="checkbox" name="<%= milestone.name %>"
                  data-color="orangered"
-                 data-remotename="<%= milestone.remoteName %>"
+                 data-remotename=milestone-"<%= milestone.name %>"
```

Comment:
This results in:

```
<input class="wc-CategoryEditor-list-item-checkbox" 
            name="worksforme" 
            data-color="orangered" 
            data-remotename="milestone-"worksforme"" 
            tabindex="-1" 
type="checkbox">
```

Note the extra quotes. It works, but could we change it to something like:

`"data-remotename=milestone-<%= milestone.name %>"` (moving the opening quote before `milestone`)